bin/pycbc_live

```diff
@@ -523,24 +563,31 @@ with ctx:
                             results[ifo][key] = results[ifo][key][idx]
 
             # Look for coincident triggers and do background estimation
-            coinc_results = {}
+            coinc_results = []
             if args.enable_background_estimation:
-                coinc_results = estimator.add_singles(results, data_reader)
-                evnt.check_coincs(results.keys(), coinc_results,
-                                  psds, args.low_frequency_cutoff, data_reader, bank,
-                                  followup_ifos=usable_followup_ifos)
+                for c in estimators:
+                    coinc_results.append(c.add_singles(results, data_reader))
```

Comment:
Will the add_singles method just ignore the ifos that do not concern it?

Reply:
yes, that's right. It now knows which ifos it is supposed to process.